agent/tool-scripts/perf

```diff
@@ -146,7 +146,7 @@ case "$mode" in
 			error_log "$script_name: start -- unable to create directory $tool_output_dir"
 			exit 1
 		fi
-	        echo "$tool_cmd" > $tool_cmd_file
+		echo "$tool_cmd" > $tool_cmd_file
```

Comment:
Is there a reason for the whitespace changes?

Reply:
@Anisha1234 ?

Reply:
@portante I tried to reindent the file and these particular lines. But I ended up creating more mess.

Reply:
Okay, so I pulled the file directly to verify that you are fixing the whitespace from spaces to tabs as it should be, thanks!